app/controllers/alaveteli_pro/info_request_batches_controller.rb

```diff
@@ -0,0 +1,76 @@
+# -*- encoding : utf-8 -*-
+# app/controllers/alaveteli_pro/info_request_batches_controller.rb
+# Controller for batch info requests
+#
+# Copyright (c) 2008 UK Citizens Online Democracy. All rights reserved.
+# Email: hello@mysociety.org; WWW: http://www.mysociety.org/
+
+class AlaveteliPro::InfoRequestBatchesController < AlaveteliPro::BaseController
+  def new
+    @draft_info_request_batch = load_draft
```

Comment:
I was doing some reading about this and decided not based on the arguments in: http://craftingruby.com/posts/2015/05/31/dont-use-before-action-to-load-data.html - as a controller file gets bigger, it's harder and harder to find the filters and check where exactly they're being used, and it's more stuff you have to keep in your head as you're looking at the rest of the controller. I didn't manage to match up exactly to what the author says, but at least the logic of what's getting assigned is in-flow rather than separately. 

All that said, I'm not particular wed to the idea, so if you hate it, I'd be happy to put them back into filters :)